pkg/backup/actions/csi/pvc_action_test.go

```diff
@@ -175,29 +176,29 @@ func TestExecute(t *testing.T) {
 					var vsList v1.VolumeSnapshotList
 					err := wait.PollUntilContextTimeout(context.Background(), 1*time.Second, 10*time.Second, true, func(ctx context.Context) (bool, error) {
 						err = pvcBIA.crClient.List(ctx, &vsList, &crclient.ListOptions{Namespace: tc.pvc.Namespace})
-
-						require.NoError(t, err)
+						//nolint:testifylint // false-positive
```

Comment:
this line should not be needed

running locally, give me an error `directive `//nolint:testifylint // false-positive` is unused for linter "testifylint" (nolintlint)`

Reply:
`golangci-lint has version v1.59.1 built with go1.22.3`